src/main/java/bio/terra/app/controller/AdminApiController.java

```diff
@@ -28,7 +37,10 @@ public class AdminApiController implements AdminApi {
   private final AuthenticatedUserRequestFactory authenticatedUserRequestFactory;
   private final DrsService drsService;
   private final IamService iamService;
+  private final DatasetService datasetService;
+  private final SnapshotService snapshotService;
   private final ApplicationConfiguration appConfig;
+  private final Logger logger = LoggerFactory.getLogger(AdminApiController.class);
```

Comment:
logger can be `static`
```suggestion
  private static final Logger logger = LoggerFactory.getLogger(AdminApiController.class);
```